core/templates/pages/admin-page/services/admin-router.service.ts

```diff
@@ -41,6 +48,7 @@ export class AdminRouterService {
 
   /**
    * Navigates the page to the specified tab based on its HTML hash.
+   * @param {string} tabHash Represents hash name of a tab URL.
```

Comment:
`tabHash The string after the '#' character in the URL.`

Reply:
Done